docs/FAQ.md

```diff
@@ -9,41 +11,102 @@ of the PV will create an FSx for OpenZFS snapshot of the file system's root volu
 increasing the requested storage request on a PVC bound to a PV that points at an FSx for OpenZFS file system will increase the storage capacity
 of the overarching file system.
 
-### How do I request storage when dynamically provisioning a new FSx for OpenZFS volume as a persistent volume in my cluster?
-When dynamically provisioning a new FSx for OpenZFS volume, the FSx for OpenZFS CSI driver ignores the storage requested in the Persistent Volume Claim (PVC).
-Instead, the driver configures storage requirements via the storageCapacityReservation and storageCapacityQuota parameters, which are defined in the storage class.
-To minimize operator confusion, **the driver will fail to create a volume if the user specifies a value other than the default 1Gi**.
+## Parameters
 
-### What happens when I expand the size of my Persistent Volumes?
-You may expand the size of a PV bound to an FSx for OpenZFS file system by increasing the requested storage in the PVC.
-This will increase the size of the file system's storage capacity to the value requested in the PVC.
-Currently, there is no way to adjust the storage requirements of a PV bound to an FSx for OpenZFS volume via the CSI driver.
-To do this, you will need to update the volume via the [FSx API](https://docs.aws.amazon.com/cli/latest/reference/fsx/index.html#cli-aws-fsx) outside the context of your cluster.
-See [here](../examples/kubernetes/volume-expansion/README.md) for more details.
+### Why is SkipFinalBackupOnDeletion required by the CSI Driver, and not the API?
+The default behavior of the API is to set this value as false, which thus creates a resource after deletion.
+To prevent users from creating dangling resources that the driver doesn't maintain, we force users to input this value.
+We recommend that this value is set to `true` so no manual action is necessary to maintain this resource.
```

Comment:
I don't think we should leave a recommendation here. Skipping the final backup is fairly risky from a customer perspective (which is why the user has to click through like 3 confirmations in the console if they want to do it there). I'd just leave this as a warning that creating a final backup will leave a dangling resource.